plugins/org.mozilla.Firefox.Plugin.Flash.yaml

```diff
@@ -0,0 +1,39 @@
+app-id: org.mozilla.Firefox.Plugin.Flash
+runtime: org.mozilla.Firefox
+sdk: org.freedesktop.Sdk//18.08
+build-extension: true
+finish-args:
+  - '--require-version=1.5.0'
+build-options:
+  prefix: /app/firefox/plugins/Flash
+  append-path: /app/firefox/plugins/Flash/bin
+  append-ld-library-path: /app/firefox/plugins/Flash/lib
+  append-pkg-config-path: /app/firefox/plugins/Flash/lib/pkgconfig
+modules:
+  - ../shared-modules/gtk2/gtk2.json
+
+  - name: flash
+    buildsystem: simple
+    build-commands:
+      - 'install -Dm 755 -t ${FLATPAK_DEST}/bin apply_extra'
+      - 'mkdir -p ${FLATPAK_DEST}/plugins'
+      - 'ln -s ${FLATPAK_DEST}/extra/libflashplayer.so ${FLATPAK_DEST}/plugins'
+    sources:
+      - type: script
+        dest-filename: apply_extra
+        commands:
+          - 'tar xf flash_player_npapi_linux.tar.gz libflashplayer.so'
+          - 'rm flash_player_npapi_linux.tar.gz'
+          - 'chmod +x libflashplayer.so'
+      - type: extra-data
+        filename: flash_player_npapi_linux.tar.gz
+        only-arches: [x86_64]
+        url: https://fpdownload.adobe.com/pub/flashplayer/pdc/32.0.0.207/flash_player_npapi_linux.x86_64.tar.gz
```

Comment:
This points to a broken URL - see https://get.adobe.com/flashplayer/webservices/json/?platform_type=Linux&platform_arch=x86-64&eventname=flashplayerotherversions&browser_dist=Firefox for latest.